tests/pyapi/test_dependencies.py

```diff
@@ -25,8 +25,8 @@ def test_dep_failed(hq_env: HqEnv):
     (job, client) = prepare_job_client(hq_env, with_worker=True)
 
     t1 = job.program(args=bash("exit 1"))
-    t2 = job.program(args=bash("echo 'hello' > foo1.txt"), depends=[t1])
-    job.program(args=bash("echo 'hello' > foo2.txt"), depends=[t2])
+    t2 = job.program(args=bash("echo 'hello' > foo1.txt"), dependencies=[t1])
+    job.program(args=bash("echo 'hello' > foo2.txt"), dependencies=[t2])
```

Comment:
Later I would vote to shorten `dependencies` to `deps`, as it will probably be used quite a lot.